tests/smoke/smoke_test.go

```diff
@@ -185,13 +178,15 @@ var _ = Describe("os2 Smoke tests", func() {
 				)
 
 				Eventually(func() string {
-					out, _ := s.Command("KUBECONFIG=/etc/rancher/k3s/k3s.yaml helm -n cattle-rancheros-operator-system install --create-namespace rancheros-operator /usr/local/ros.tgz")
+					_, _ = s.Command("KUBECONFIG=/etc/rancher/k3s/k3s.yaml helm repo add rancheros https://rancher-sandbox.github.io/rancheros-operator/")
```

Comment:
My guess here would be to drop this test part entirely (setting up k3s and the operator chart) as we would have separated e2e tests in the chart repo, no? at the end here was about having a small test to see if the ros chart is functional. 

I'd just keep around the k3s part to see if a cluster is functional, without need to test our chart

Reply:
yes, I just want to see if this work with the os2 iso or I broke something. I guess this is the best chance I got to see if something is missing or broken :+1: